### ISSUE TITLE
Infighting - Goblin - should be racial modifier, not bonus.

### ISSUE THREAD

**Issue by klondikemarlen** [elthran/UndyingKingdoms#239]
This is on the economy page, the tooltip for infighting should say Racial Modifier not bonus as it is a handicap.

**Comment by klondikemarlen** [comment]
Fixed on Desktop version of site in pull request https://github.com/elthran/UndyingKingdoms/pull/255

**Comment by elthran** [comment]
I agree. This is important for flavour and clarity. I will put the UI guy in charge of this

**Comment by elthran** [comment]
Is this done? I think it now just says 'Racial Modifier:' which seems perfect.

**Comment by klondikemarlen** [comment]
It is still bugged on mobile. This page is particularly out of sync.
